.github/ISSUE_TEMPLATE/task.yml

```diff
@@ -0,0 +1,37 @@
+name: "Task Report:  Groupware"
+description: "Create new meta task"
+title: "[Task]: "
+labels: ["task", "1. to develop"]
+assignees:
+  - octocat
+body:
+  - type: textarea
```

Comment:
I would prefer markdown

Reply:
if we do the task list as a mark down theres no place where to write the list. Am i missing smth? 
Check how it looks with textarea: https://github.com/GretaD/dummy-repository/blob/main/.github/ISSUE_TEMPLATE/task.yml

Reply:
okay, then it might need a `render: markdown` in addition to `type: textarea`

Reply:
this still doesn't render as markdown: https://github.com/GretaD/dummy-repository/issues/28#issue-1122018047